sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -209,7 +209,7 @@ the Azure CLI's client ID will be used.
   `client_id`
   - transport configuration is now done through keyword arguments as
   described in
-  [`azure-core` documentation](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/docs/configuration.md)
+  [`azure-core` documentation](https://github.com/Azure/azure-sdk-for-python/blob/azure-identity_1.0.0b4/sdk/core/azure-core/docs/configuration.md)
```

Comment:
```suggestion
  [`azure-core` documentation](https://github.com/Azure/azure-sdk-for-python/blob/azure-identity_1.0.0/sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md#transport)
```

Reply:
This new link doesn't cover transport customization. The one I suggested does, and I believe it's correct for azure-identity 1.0.0b4.